src/core_plugins/timelion/public/webpackShims/jquery-flot.js

```diff
@@ -1,7 +1,16 @@
+const $ = require('jquery');
+
+if (window) window.jQuery = $;
```

Comment:
You're right, it does, and it's why making it into a shim was a requirement. In the webpack context, `window` is shimmed, and doesn't include anything. You're meant to modify it like this. When used in the browser, you end up clobbering any other version of jQuery already on window, and that's what caused issues.